lib/DocumentsFieldArray/DocumentField.js

```diff
@@ -86,7 +86,7 @@ export default class DocumentField extends React.Component {
             ]}
             id={`${id}-category`}
             label={<FormattedMessage id="stripes-erm-components.doc.category" />}
-            name={`${name}.atType`}
+            name={`${name}.atType.value`}
```

Comment:
I'm not certain about this. We normally don't use this approach because [we process the record before providing it as `initialValues`.](https://github.com/folio-org/ui-licenses/blob/9e73260a51b06c2d154ec84bb9ce4ac040dddd59/src/routes/EditLicenseRoute.js#L130). 

I'm assuming this fix will only work in Agreements but will break Licenses. If so, can you instead fix this in Agreements by transforming the docs objects (that have doc types) instead?